### ISSUE TITLE
Game crashes at welcome message, enabling virtual desktop within winecfg fixed it.

### ISSUE THREAD

**Issue by Redecorating** [Linux74656/SpaceEngineersLinuxPatches#46]
My game was crashing when it loads the menu.
I fixed it with
`WINEPREFIX=~/.local/share/Steam/steamapps/compatdata/244850/pfx/ winecfg` 
And in the graphics tab I enabled `Emulate a virtual desktop`. This has gotten the game to the menu both times that I've tried it, I haven't tried starting a game, as I'm quite busy, but I'll try that soon.
I suppose using a virtual desktop could be added to the list of fixes?

**Comment by Linux74656** [comment]
Change made.

**Comment by Xinef** [comment]
I've had the same issue, and found out a few things:
1. While the fix with emulating a virtual desktop stops the game from crashing, it doesn't like multi-monitor setups. The game would move to occupy only the right half of my primary screen, the other half being inaccessible (my secondary screen is to the left of the primary).
2. While I could simply turn off the secondary screen to fix this, I decided to look for other solutions.
3. I found out that the game only crashes in fullscreen mode. Switching to windowed, or fullscreen window mode resolves the crashing, at which point virtual desktop emulation is no longer needed.

Oh, and one other observation - the size of the emulated virtual desktop did not matter, the game would always resize the desktop to the resolution of the primary screen, but then move half the screen to the right.
